src/main/java/sirius/tagliatelle/rendering/GlobalRenderContext.java

```diff
@@ -35,6 +38,27 @@
     protected Map<String, String> extraBlocks;
     protected Function<String, String> escaper = GlobalRenderContext::escapeRAW;
 
+    /**
+     * Cookie name to enable debugging of rendered contents
+     */
+    public static final String SIRIUS_DEBUG_COOKIE = "SIRIUS.WEB.DEBUG.LEVEL";
+
+    /**
+     * Debug levels used to render contents
+     */
+    public enum DebugLevel {
+        OFF, DEBUG, TRACE
+    }
+
+    // Stores the current debug level
```

Comment:
use javadoc syntax instead of normal comments

Reply:
Converted to JavaDoc.